Author: Alastair McKinstry <mckinstry@debian.org>
Description: Drop refs to files that don't exist
Last-Updated: 2016-10-27
Forwarded: no

Index: eccodes-2.21.0/configure.ac
===================================================================
--- eccodes-2.21.0.orig/configure.ac
+++ eccodes-2.21.0/configure.ac
@@ -39,7 +39,7 @@ AM_SANITY_CHECK
 AC_CONFIG_SRCDIR([src/grib_api.h])
 AC_CONFIG_HEADER([src/config.h])
 AC_CONFIG_FILES([src/eccodes_version.h])
-AC_CONFIG_FILES([rpms/eccodes.pc rpms/eccodes.spec rpms/eccodes_f90.pc])
+# AC_CONFIG_FILES([rpms/eccodes.pc rpms/eccodes.spec rpms/eccodes_f90.pc])
 AM_INIT_AUTOMAKE($PACKAGE_NAME,[${PACKAGE_VERSION}],[http://www.ecmwf.int])
 
 definition_files_path="${datadir}/eccodes/definitions"
